README.md

```diff
@@ -76,6 +76,29 @@ And this is what gets rendered (plus the CSS and Javascript you've specified):
 
 ## Release notes
```

Comment:
I'm getting to the point for me where having the full release notes inlined in the README is making it hard to read. So as one of the MRs for the `dev`, I would like to move the release notes to their own page, and just link to them from here.